package.json

```diff
@@ -42,7 +43,7 @@
     "jest": "^24.8.0"
   },
   "scripts": {
-    "build": "babel -d . src/ --ignore '**/*.test.js'",
+    "build": "babel -d . src/ --ignore '**/*.test.js' && cp src/*.d.ts .",
```

Comment:
Not sure if it is reasonable to expect `cp` to be available at buildtime. If not, we can use some node dev dependency to copy the files to root.